modules/kmm-customizing-upgrades-for-kernel-modules.adoc

```diff
@@ -37,15 +36,30 @@ The update should be atomic; that is, both the `containerImage` and `version` fi
 
 . Terminate any workload using the kernel module on the node being upgraded.
 
-. Remove the `kmm.node.kubernetes.io/version-module.<module-namespace>.<module-name>` label on the node. This unloads the kernel module from the node.
+. Remove the `kmm.node.kubernetes.io/version-module.<module-namespace>.<module-name>` label on the node.
+This unloads the kernel module from the node:
++
+[source,terminal]
+----
+$ oc label node/<node_name> kmm.node.kubernetes.io/version-module.<module_namespace>.<module_name>-
+----
 
-. If required, perform any additional maintenance required on the node for the kernel module upgrade.
+. If required, as the cluster administrator, perform any additional maintenance required on the node for the kernel module upgrade.
++
+If no additional upgrading is needed, you can skip Steps 3 through 6 by updating the `kmm.node.kubernetes.io/version-module.<module-namespace>.<module-name>` label value to the new `$moduleVersion` as set in the `Module`.
```

Comment:
```suggestion
If no additional upgrading is needed, you can skip Steps 3 through 6 by updating the `kmm.node.kubernetes.io/version-module.<module_namespace>.<module_name>` label value to the new `$moduleVersion` as set in the `Module`.
```
Since this is Step 5, the note that you can "skip Steps 3 through 6 if no additional upgrading is needed" is out of sequence. Should we skip steps 6-9?

Reply:
No change here.